app/service/log_prefixer.go

```diff
@@ -0,0 +1,71 @@
+package service
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+)
+
+const prefixCommandMaxlen = 16
+const prefixCutCommandSuffix = "..."
+
+// LogPrefixer implements `io.writer` interface and adds prefix to each output line.
+type LogPrefixer struct {
+	writer io.Writer
+	prefix []byte
+}
+
+// NewLogPrefixer initializes log prefixer.
+func NewLogPrefixer(writer io.Writer, command string) *LogPrefixer {
+	logPrefixer := &LogPrefixer{writer: writer}
+	logPrefixer.prefix = logPrefixer.prefixForCommand(command)
+	return logPrefixer
+}
+
+func (p *LogPrefixer) Write(data []byte) (int, error) {
```

Comment:
do we really need all of this? How about this:

```
if len(data) >0 {
 p.Write(p.prefix)
 _, err := p.Write(data)
 return len(data), err
}
```

Reply:
But it will only prefix only first line of the command output.

Reply:
It would probably work if command produces output line by line, means internally calling `Write` for every line. Which is not the case even for `ls -l`, just tried.

```
2020/07/28 02:05:05 INFO  executing: "ls -l"
{ls -l} total 17576
-rw-r--r--  1 egor  staff      650 Jul 26 14:40 Dockerfile
-rw-r--r--  1 egor  staff     1842 Jul 26 14:32 Dockerfile.artifacts
-rw-r--r--  1 egor  staff     1064 Jul 26 14:30 LICENSE
...
2020/07/28 02:05:05 INFO  completed ls -
```